### PR TITLE
feat: support files for fs.allow #411

### LINKED ISSUE #411: feat: support files for `fs.allow`
New updates on head repo.
https://github.com/vitejs/vite/commit/4a06e66130ee4701b9461081ab09ad88cadc5ac2

### PR DESCRIPTION
**Pull request by marcterradas**
Resolves #411

### CODE REVIEW

**Comment by mrcego** [comment]
@marcterradas

Check please function example from #L283,   I'm almost sure something is missing.

File: docs/config/server-options.md [hunk 1]
```diff
@@ -288,6 +290,8 @@ createServer()
           searchForWorkspaceRoot(process.cwd()),
           // tus reglas personalizadas
           '/path/to/custom/allow',
```
Comment (mrcego) [inline docs/config/server-options.md:292]: This line should be removed according issue.

**Review by mrcego — CHANGES_REQUESTED** [review]

File: docs/config/server-options.md (hunk 1 above)
Comment (marcterradas) [reply docs/config/server-options.md]: yes! sorry 👍

**Comment by mrcego** [comment]
Thanks.

**Pull request merged**